src/app/shared/components/template/template-container.component.ts

```diff
@@ -224,13 +206,15 @@ export class TemplateContainerComponent implements OnInit, OnDestroy, ITemplateC
 
   /** Query params are used to track state across navigation events */
   private subscribeToQueryParamChanges() {
-    this.route.queryParams
-      .pipe(takeUntil(this.componentDestroyed$))
-      .subscribe(async (params: any) => {
-        this.debugMode = params.debugMode ? true : false;
-        // allow templateNavService to process actions based on query param change
-        await this.templateNavService.handleQueryParamChange(params, this);
-      });
+    if (this.route) {
```

Comment:
another catch added for cases where we are processing a template in standalone mode - we do not have access nor want to subscribe to the changes to the currently active url ( `route`)